terraform/templates/ec2_user_data.sh

```diff
@@ -50,24 +50,45 @@ export PS1="[\u:validator@\h \w]$ "
 EOF
 
 {% if enable_logrotate %}
+{% if persistent %}
 cat > /etc/logrotate.d/libra <<EOF
 hourly
-${host_log_path} {
```

Comment:
Would suggest keeping the config consistent and avoid boiler plate.  Can we move the change to the `if ! ${persistent}` block in line 17 above?

Reply:
true, that would be cleaner. updated!